iam/clouds/aws/CloudGovernanceInfra/README.md

```diff
@@ -0,0 +1,80 @@
+## Create CloudGovernance Infra in the cloud
+
+#### Requirements
+
+- IAM User: to access cloud resources.
+- IAM Policy: Least privilege principle
+- S3-Bucket: To store the logs of cloud-governance policy runs.
+
+### Pre-requisites
+
+- Install [AWS-CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+- Configure IAM Access credentials.
+    - Permissions required for the User.
+    ```json lines
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Sid": "STS",
+            "Effect": "Allow",
+            "Action": "sts:GetCallerIdentity",
+            "Resource": "*"
+          },
+          {
+            "Sid": "IAMPermissions",
+            "Effect": "Allow",
+            "Action": [
+              "iam:CreateUser",
+              "iam:CreatePolicy",
+              "iam:DeleteUser",
+              "iam:DeletePolicy",
+              "iam:AttachUserPolicy",
+              "iam:CreateAccessKey"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Sid": "S3Bucket",
+            "Effect": "Allow",
+            "Action": [
+              "s3:CreateBucket",
+              "s3:DeleteBucket"
+            ],
+            "Resource": "*"
+          }
+        ]
+  }
+  ```
+
+Steps to create Cloud Governance Infra resources:
+
+- Download tar `CloudGovernanceInfra.tar` and untar the file.
+
+```shell
+curl -L https://github.com/redhat-performance/cloud-governance/raw/main/iam/clouds/aws/CloudGovernanceInfra.tar | tar -xzvf -
+```
+
+- Change the permissions of the file `create_infra.sh`.
+
+```shell
+chmod 700 create_infra.sh
+```
+
+- Create read user and create bucket.
+
+```shell
+./create_infra.sh --username "$USERNAME" --s3-bucket-name "$BUCKET_NAME"
+```
+
+- Create only delete user
+
+```shell
+./create_infra.sh --username "$USERNAME" --s3-bucket-name "$BUCKET_NAME" --policy-type delete
+```
+
+- To get help
+
+```shell
+./create_infra.sh --help
+```
```

Comment:
Do you want to add image that the explain the process ?

Reply:
./create_infra.sh --username "$USERNAME" --s3-bucket-name "$BUCKET_NAME" --policy-type delete
Dont need to run 
terraform destroy